From e0d6005e7bedd06099201f8be34959b160cf235c Mon Sep 17 00:00:00 2001
From: Jim Fehlig <jfehlig@suse.com>
Date: Tue, 5 Jul 2022 13:52:35 -0600
Subject: libxl: Use the SUSE ovmf firmware path for Xen

The libxl driver trivially supports firmware autoselection since as of
June 2021 ovmf only supports one firmware for Xen. This patch adjusts
the firmware path to match the one provided by the ovmf package.

Signed-off-by: Jim Fehlig <jfehlig@suse.com>
---
 src/libxl/libxl_conf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libvirt-8.7.0/src/libxl/libxl_conf.c
===================================================================
--- libvirt-8.7.0.orig/src/libxl/libxl_conf.c
+++ libvirt-8.7.0/src/libxl/libxl_conf.c
@@ -1788,7 +1788,7 @@ libxlDriverConfigNew(void)
     cfg->firmwares = g_new0(virFirmware *, 1);
     cfg->nfirmwares = 1;
     cfg->firmwares[0] = g_new0(virFirmware, 1);
-    cfg->firmwares[0]->name = g_strdup(LIBXL_FIRMWARE_DIR "/ovmf.bin");
+    cfg->firmwares[0]->name = g_strdup(DATADIR "/qemu/ovmf-x86_64-xen-4m.bin");
 #endif
 
     /* Always add hvmloader to firmwares */
